src/web/editor.template.html

```diff
@@ -104,7 +104,6 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
 <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
 <script src="/js/loader.js"></script>
-<script src="/js/url.js"></script>
```

Comment:
There isn't anything here that `pyret-lang` depends on to run the compiler/language runtime.  Actually, since `world` and `image` are now in the CPO repo as well, the situation is actually better than it was, dependency-wise.

So any uses of globals defined in these script tags will be within the CPO repository, not lurking elsewhere, at least.  I'll write up more dependency information in a comment on the PR as a whole.